.travis.yml

```diff
@@ -0,0 +1,32 @@
+os: linux
+dist: xenial
+language: python
+python:
+  # We don't actually use the Travis Python, but this keeps it organized.
+  - "3.7"
+env:
+  global:
+    - DRAGONS_TEST="$HOME/dragons_tests/"
+    - JENKINS_CONDA_HOME="$HOME/miniconda"
```

Comment:
Not needed ?

Reply:
`DRAGONS_TEST` is the root folder where the files downloaded by the `download_from_archive` function will be cached. Removed `JENKINS_CONDA_HOME`.